app/react/package.json

```diff
@@ -19,12 +19,14 @@
   },
   "scripts": {
     "dev": "DEV_BUILD=1 nodemon --watch ./src --exec 'npm run prepublish'",
-    "prepublish": "node ../../scripts/prepublish.js"
+    "prepublish": "node ../../scripts/prepublish.js",
+    "test": "../../node_modules/.bin/jest --watch"
   },
   "dependencies": {
     "@storybook/addon-actions": "^3.1.6",
     "@storybook/addon-links": "^3.1.6",
     "@storybook/addons": "^3.1.6",
+    "@storybook/addon-options": "^3.1.6",
```

Comment:
FWIW i agree with @igor-dv that `addon-options` is a strange package and we should consider moving the functionality into `addons`. I've opened an issue https://github.com/storybooks/storybook/issues/1430

I believe this is complementary to my proposal lower down to create a `storiesOf(...).setOptions({ ... })` method for configuring showing multiple preview windows.